README.md

```diff
@@ -272,6 +272,13 @@ In this release, we have abstracted these implementation details away and expose
 
 ## Release Notes
 
+### Release 0.7.0 (August 2, 2017)
+* Updated the dependency on [Amazon Kinesis Client for Java][amazon-kcl-github] to 1.8.1.
+* Support for timeout calls to MultiLang Daemon.
```

Comment:
I wouldn't include a bullet point for this, just put all this under the previous bullet point.